src/basilisp/io.lpy

```diff
@@ -31,7 +32,13 @@
   urllib.parse/ParseResult
   (as-path [f]
     (if (contains? #{"file" ""} (.-scheme f))
-      (-> (.-path f) (pathlib/Path))
+      (let [path (.-path f)]
+        (if (= sys/platform "win32")
```

Comment:
I filed #692 to eventually support having a reader feature for this so we don't have to switch on it at runtime.

Reply:
Sounds good